src/main/java/frc/robot/Robot.java

```diff
@@ -44,6 +50,17 @@ public void robotPeriodic() {
     // and running subsystem periodic() methods.  This must be called from the robot's periodic
     // block in order for anything in the Command-based framework to work.
     CommandScheduler.getInstance().run();
+
+    // REMEMBER TO MOVE THIS OUT OF ROBOT PERIODIC
+    // Logs the Robot Data
+    if(m_logTime >= m_logInterval) {
+      m_logTime = 0;
+      m_logger.updateFields();
+      m_logger.log();
+    } else {
+      m_logTime += 20;
```

Comment:
Rather than incrementing this by a fixed `20` here, should we increment by the elapsed time since the last execution?